Sources/SwiftMemcache/MemcachedRequest.swift

```diff
@@ -11,3 +11,20 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 //===----------------------------------------------------------------------===//
+
+import NIOCore
+
+enum MemcachedRequest {
+    struct SetCommand {
+        let key: String
+        var value: ByteBuffer
+    }
+    case set(SetCommand)
+
+    var command: String {
+        switch self {
+        case .set:
+            return "ms"
+        }
+    }
```

Comment:
This still allocates a `String`. I would recommend to just remove this computed property and write out the bytes directly similar to how you do whitespace.